src/js/utils/ContainerValidatorUtil.js

```diff
@@ -0,0 +1,13 @@
+const ContainerValidatorUtil = {
+  isValidDockerImage(dockerImage) {
+    if (typeof dockerImage !== 'string' || dockerImage === '') {
+      return false;
+    }
+
+    // The pattern is based on Dockers `validContainerName` regexp:
+    // https://github.com/docker/docker/blob/f63cdf0260/runtime.go#L33
+    return /^[a-zA-Z0-9_\-/:.]+$/.test(dockerImage);
```

Comment:
It's based (!) on the `validContainerName` pattern but also allows `/`,`:`, and `.` which are used in the other parts (user/org and tag) of a valid docker image specification.